src/ui/ImageViewer.css

```diff
@@ -0,0 +1,5 @@
+.widget-image-viewer img {    
+    max-width: 100%;
+    max-height: 100%;
+    object-fit: contain;
```

Comment:
Does not work IE nor Edge https://developer.mozilla.org/nl/docs/Web/CSS/object-fit#Browser_compatibility